x-pack/legacy/plugins/snapshot_restore/public/shared_imports.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export {
+  SendRequest,
+  SendRequestResponse,
+  UseRequest,
+  sendRequest,
+  useRequest,
+} from '../../../../../src/plugins/elasticsearch_ui_shared/public';
```

Comment:
I don't feel super strongly about this but maybe the shared directory could be called `es_ui_shared` instead to shorten the path? cc @sebelga as you were working in this shared directory too

Reply:
No preference on then naming 😊 `es_ui_shared` sounds more catchy. It can be interpreted as "elasticsearch ui" or "spanish ui" so I'm good with it!